app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,11 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+      <div class="container">
```

Comment:
Anh nhắn là phải kiểm tra hết các phần trong checklist r mới to mà nhỉ. Sao đây vẫn còn sai indent??